clients/githubrepo/client.go

```diff
@@ -259,8 +261,33 @@ func CreateGithubRepoClientWithTransport(ctx context.Context, rt http.RoundTripp
 	httpClient := &http.Client{
 		Transport: rt,
 	}
-	client := github.NewClient(httpClient)
-	graphClient := githubv4.NewClient(httpClient)
+
+	var client *github.Client
+	var graphClient *githubv4.Client
+	// check if GITHUB_API_URL is set
+	if githubAPIURL := os.Getenv("GITHUB_API_URL"); githubAPIURL != "https://api.github.com" {
+		// create a new enterprise client with custom URLs
+		githubSERVERURL := os.Getenv("GITHUB_SERVER_URL")
+		if githubSERVERURL == "" {
+			// load the server url from the api url
+			githubSERVERURL = strings.TrimSuffix(githubAPIURL, "/api/v3")
+		}
+		// trim trailing slash to prevent issues with the graphql client
+		githubSERVERURL = strings.TrimSuffix(githubSERVERURL, "/")
+		githubGRAPHQLURL := fmt.Sprintf("%s/api/graphql", githubSERVERURL)
```

Comment:
This line didn't work for me.  To run the scorecards, I used `GITHUB_API_URL=https://api.github.ibm.com` and changed this line to

```golang
githubGRAPHQLURL := fmt.Sprintf("%s/graphql", githubSERVERURL)
```

Using /api/graphql I caused 404s, using /graphql allowed the scorecard to generate.  I only have access to my own company's GHE, so I don't know if you should always use /graphql or if this needs to be parameters for different configurations.